packages/interactivity/src/utils.ts

```diff
@@ -22,6 +22,14 @@ interface Flusher {
 	readonly dispose: () => void;
 }
 
+declare global {
+	interface Window {
+		scheduler: {
+			readonly yield: () => void;
```

Comment:
```suggestion
			readonly yield: () => Promise<void>;
```
Maybe consider making scheduler and yield optional, too, so anyone coming along later will be forced to feature detect?

Reply:
Good idea. Done in 3d40b9f37427f4bc309330853bcb6394def37e08.